src/components/ShoppingList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../lib/firebase';
+
+const ShoppingList = () => {
+  const [term, setTerm] = useState('');
+
+  const [value, loading, error] = useCollection(
+    db.collection('shoppingLists'),
+    {
+      snapshotListenOptions: { includeMetadataChanges: true },
+    },
+  );
+
+  const onInputChange = (e) => {
```

Comment:
If we have multiple inputs the name of this function becomes unclear. Additionally it would be beneficial to follow the name of the state being updated and the item the input is accepting. Having these three things with similar names helps understand the code flow better.

Reply:
Changed it to "onGroceryItemChange" to better reflect the state that is being changed

Reply:
This definitely fixes the descriptor issue. We may become confused however if we allow editing of an existing item, as the handler for that change could be named `onGroceryItemChange`. Since I think `input` is an important descriptor as well, may I propose `onGroceryItemInputChange`? It's on the more verbose side of what I'd recommend and is certainly debatable, but it describes both what (the input for a groceryItem) and when (onChange). Feel free to push back on this if you disagree!

Reply:
That makes sense to me! It's better to be verbose than to cause confusion - good point! Updated it to be 
`onGroceryItemInputChange`